src/routes/tokens/component/Layout.jsx

```diff
@@ -49,11 +49,23 @@ class TokenLayout extends React.PureComponent<TokenProps, State> {
     })
   }
 
-  /*
-    onRemoveToken = () => {
-      this.setState({ component: <RemoveToken /> })
-    }
-  */
+  onReset = () => {
+    this.setState({ component: undefined })
+  }
+
+  onRemoveToken = (token: Token) => {
+    const { safeAddress, removeToken } = this.props
+
+    this.setState({
+      component: <RemoveToken
```

Comment:
Is this a good practice?
http://web.archive.org/web/20150419023006/http://facebook.github.io/react/docs/interactivity-and-dynamic-uis.html#what-shouldnt-go-in-state

Reply:
No, I do not think it is a good practice, in general, state should contain the info of component's "natural life (data fetched, answers to user interactions...)", but  the way I coded the token and safe views were thinking in something like a desktop app, where, in one corner based on the operation I load the form for each one.

This is going to change soon UX/UI designs are WIP, if the distribution of the UI does not change, I will refactor it and save in the redux' store the user action, and then using some logic in the render method I will load the correct form.

Thanks for the comment.